xds/src/main/java/io/grpc/xds/XdsServerBuilder.java

```diff
@@ -41,10 +46,17 @@
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/7514")
 public final class XdsServerBuilder extends ForwardingServerBuilder<XdsServerBuilder> {
 
+  public static final Attributes.Key<AtomicReference<FilterChainSelector>>
+          ATTR_FILTER_CHAIN_SELECTOR_REF = Attributes.Key.create(
+          "io.grpc.xds.XdsServerBuilder.filterChainSelectorRef");
+  private static final long RETRY_DELAY_NANOS = TimeUnit.MINUTES.toNanos(1);
```

Comment:
Can you just move this to the `XdsServerWrapper` class so you don't need the `retryDelayNano` param to the constructor? I don't see that param being used or even tested in any unit tests